src/core/core.scale.js

```diff
@@ -359,6 +359,14 @@ export default class Scale extends Element {
     return this.options.labels || (this.isHorizontal() ? data.xLabels : data.yLabels) || data.labels || [];
   }
 
+  /**
+   * @return LabelItem[]
+   */
+  getLabelItems(chartArea) {
```

Comment:
User should not be needed to supply the chart area, depending on implementation it either should provide nothing if it calls it on the scale itself or pass the the scaleId that it wants the labelItems from

Reply:
@LeeLenaleee It would seem that way, but I'm not sure - the chartArea is passed all the way in from the .draw() function, and I don't have enough knowledge right now to say that the chartArea will always be derived from the chart.chartArea without changes.

Reply:
However, it could easily be made optional:
`getLabelItems(chartArea = this.chart.chartArea) {`